EnglishTranslation-master/ITEM.tsv

```diff
@@ -9818,18 +9818,18 @@ ITEM_20151224_009817	Spell Amplifier
 ITEM_20151224_009818	An amplifier carried by Black Maizes. An item wanted by Revelator Mihail.
 ITEM_20151224_009819	Raffly Soul Crystal
 ITEM_20151224_009820	Vubbe Warrior Mind Crystal
-ITEM_20160224_009821	Many followed suit once one of the heirs of the more prestigious families fashioned their hair this way. It must be said however, that it is difficult to diverge all of its charm without the person being attractive in the first place.
-ITEM_20160224_009822	A hairstyle restored around the time of the Millenary which many people tried to imitate. However, it was mostly done by the nobility because it is difficult to cut and maintain.
+ITEM_20160224_009821	Once a heir of one the more prestigious families fashioned their hair this way, many followed suit. It must be said however, it's difficult to capitalize on its charm without being attractive in the first place.
+ITEM_20160224_009822	A hairstyle imitated by many, restored from around the time of the Millenary. However, it was mostly done by nobility due to it being difficult to cut and maintain.
 ITEM_20160224_009823	Root Lifted Wave Hairstyle
-ITEM_20160224_009824	Some say that a mere hairstyle cannot become the starting point of showing one's respect for the goddesses.{nl}However, goddesses also have preferences.
+ITEM_20160224_009824	Some say that a mere hairstyle cannot be a way to show one's respect for the goddesses.{nl}However, goddesses also have preferences.
 ITEM_20160224_009825	Noble Ponytail
-ITEM_20160224_009826	A hairstyle that has been in fashion each time the kingdom was at peace. This hairstyle signifies the longing for those peaceful times in these troublesome times after Medzio Diena.
-ITEM_20160224_009827	According to some rumors, General Ramin was able to find the princess in the ruins of the old capital when she was half buried in debris thanks to this hairstyle.
+ITEM_20160224_009826	A hairstyle that's came into fashion each time the kingdom was at peace. This hairstyle signifies longing for the peace of times past, before Medzio Diena.
+ITEM_20160224_009827	According to some rumors, General Ramin was able to find the princess half buried in the ruins of the old capital thanks to this hairstyle.
```

Comment:
I'm not sure if this a male or female hairstyle, so if someone can clarify, I'll gender the phrase the "thanks to this hairstyle" appropriately.